### PR TITLE
[No QA]Fix iOS production deploys

### PR DESCRIPTION
**Pull request by AndrewGable**
### Details
Debugged fastlane locally and was able to submit production builds using this code. I believe production should fail once more, then it should run this code and succeed.

### Fixed Issues

$ https://github.com/Expensify/App/issues/4978

### Tests
I ran these locally, if you do not have the secrets decrypted this will not work
1. Set the version `export VERSION='1.0.92.0'
2. Run `bundle exec fastlane ios production`
3. Verify you see the build submitted for review 🎉

### CODE REVIEW

File: .github/workflows/platformDeploy.yml [hunk 1]
```diff
@@ -243,9 +243,7 @@ jobs:
 
       - name: Set iOS version in ENV
         if: ${{ fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
-        run: |
-          echo "IOS_VERSION=$(echo '${{ github.event.release.tag_name }}' | tr '-' '.')" >> $GITHUB_ENV
-          echo "iOS version is: ${{ env.IOS_VERSION }}"
+        run: echo "IOS_VERSION=$(echo '${{ github.event.release.tag_name }}' | tr '-' '.')" >> $GITHUB_ENV
```
Comment (roryabraham) [inline .github/workflows/platformDeploy.yml:246]: Can we add the echo back in for debugging purposes, or is there something functionally different here?

**Review by roryabraham — DISMISSED** [review]
Overall I don't know enough about each of these flags to really critique, but I'm really glad you were able to test locally and hopefully figure it out. Let's see if it works in the wild!

Beyond the one comment about echoing the IOS_VERSION from the GH Workflow, the only other NAB I have is that we should have a blank line above each new comment in the Fastfile, if possible. It's just a bit hard to read as-is.

File: .github/workflows/platformDeploy.yml (hunk 1 above)
Comment (AndrewGable) [reply .github/workflows/platformDeploy.yml:246]: It doesn't work using this one line syntax, I think the entire line needs to be executed, then we could run this. I verified this here: https://github.com/Andrew-Test-Org/Public-Test-Repo/actions/runs/1195174314/workflow#L13-L19

Screen Shot 2021-09-02 at 4 43 29 PM

I personally was using the ENV variables to debug this just as easily:

Screen Shot 2021-09-02 at 4 43 00 PM

Happy to add it into it's own step if you think it's valuable, I just thought it was duplicated.

Comment (roryabraham) [reply .github/workflows/platformDeploy.yml:246]: Nice, it's obviously visible there.

**Review by roryabraham — APPROVED** [review]

**Pull request merged**